server/src/metadata/object-metadata/hooks/before-create-one-object.hook.ts

```diff
@@ -21,6 +36,14 @@ export class BeforeCreateOneObject<T extends CreateObjectInput>
       throw new UnauthorizedException();
     }
 
+    if (
+      OBJECT_TYPES.includes(instance.input.nameSingular.trim().toLowerCase()) ||
+      OBJECT_TYPES.includes(instance.input.nameSingular.trim().toLowerCase())
```

Comment:
it seems you are checking the same thing twice here?

Reply:
Regarding the check itself, it will probably never match because you call toLowerCase() on the input but OBJECT_TYPES contains uppercase characters

Reply:
Note that we don't use trim in the service (nor toLowerCase) so here you are validating an input that won't 100% translate to what we will later use.